src/commonMain/kotlin/baaahs/sim/FakeNetwork.kt

```diff
@@ -47,6 +47,10 @@ class FakeNetwork(
             return FakeUdpSocket(serverPort)
         }
 
+        override fun mdns(): Network.Mdns {
+            TODO("not implemented")
```

Comment:
We'll eventually want some trivial implementation here so the simulator keeps working.